packages/strapi-connector-mongoose/lib/queries.js

```diff
@@ -551,8 +551,10 @@ module.exports = ({ model, modelKey, strapi }) => {
 
     const $or = buildSearchOr(model, params._q);
 
+    delete filters.where._q;
```

Comment:
You can use lodash `unset` `_.unset(filters, 'where._q')` to avoid a bug if the function is called without a _q param